meson_options.txt

```diff
@@ -16,3 +16,6 @@ option('read-timeout', type : 'string', value : '0')
 # the default pathname for recording shell auditable events.
 #
 option('audit-file', type : 'string', value : '/etc/ksh_audit')
+
+# Configure if api tests should be built
```

Comment:
We should start including actual examples so that people not familiar with Meson have an easier time. For example:

```
# To disable building and running non-KSH API tests configure the build thusly:
#
#   meson -Dbuild-api-tests=false
```